mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientidentifiers/ClientIdentifiersFragment.java

```diff
@@ -130,16 +132,34 @@ public void showUserInterface() {
     public void showClientIdentifiers(List<Identifier> identifiers) {
         this.identifiers = identifiers;
         identifierListAdapter.setIdentifiers(identifiers);
+        identifierListAdapter.notifyDataSetChanged();
```

Comment:
I read about `DiffUtils` and if I understood it well, it's a good option to juxtapose two lists. In our case, however, we are sure of either 'adding' or 'removing' a single identifier. How about we have a method in the adapter to add (or remove the identifier) and then `dispatchUpdatesTo()` the adapter? Or is `DiffUtils` a better option?